src/test/groovy/pl/cleankod/exchange/AccountSpecification.groovy

```diff
@@ -1,48 +1,25 @@
 package pl.cleankod.exchange
 
-import com.github.tomakehurst.wiremock.WireMockServer
-import com.github.tomakehurst.wiremock.client.WireMock
-import com.github.tomakehurst.wiremock.core.WireMockConfiguration
 import org.apache.http.HttpResponse
 import pl.cleankod.BaseApplicationSpecification
-import pl.cleankod.exchange.core.domain.Account
-import pl.cleankod.exchange.core.domain.Money
+import pl.cleankod.exchange.core.dto.AccountDto
 
 import java.nio.charset.StandardCharsets
 
 class AccountSpecification extends BaseApplicationSpecification {
```

Comment:
You removed the unit test for the integration test, why didn't you add the integration test instead?